### DISCUSSION TITLE
center align Text

### DISCUSSION THREAD

**Discussion by GambitFPS** [facebook/docusaurus#6518] [Q&A]
How do I force center aligning my text at all time, regardless of the sideBar being collapsed or not?
![grafik](https://user-images.githubusercontent.com/54640740/151721489-23c1ff28-ec3a-4fee-98ff-2123783a1355.png)

**Comment by homotechsual** [comment]
It will depend on the specifics of the site but I generally find a combination of `width: 100%` and `text-align: center` works.

**Reply by homotechsual** [reply]
However it seems like you're suggesting that when the sidebar is expanded the text is center aligned?

**Reply by GambitFPS** [reply]
I want the text to be center aligned at all time, when the sidebar is expanded as well as when it is collapsed.
(Also no, even when the sidebar is expanded the text is still not totally centered, its still a little bit too far on the left)

**Reply by GambitFPS** [reply]
> It will depend on the specifics of the site but I generally find a combination of `width: 100%` and `text-align: center` works.

Where would I have to add `width: 100%` and `text-align: center`?
